deployments/warp_routes/dreams/solanamainnet-starknet-config.yaml

```
@@ -0,0 +1,20 @@
+# yaml-language-server: $schema=../schema.json
+options: {}
+tokens:
+  - addressOrDenom: G5yLMZ1q3Xd6oPaTiuPUS3bFaPamQd1sxdYacsgB8Egp
+    chainName: solanamainnet
+    coinGeckoId: daydreams
+    connections:
```

also want the collateral address here
```suggestion
		collateralAddressOrDenom: XYZ
    connections:
```